src/tutorial/tutorial.tsx

```diff
@@ -1,11 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
-
+import Tutorials from './modal.component';
 
 const Tutorial = () => {
   const { t } = useTranslation();
+  const [isModalOpen, setModalOpen] = useState(false);
```

Comment:
```suggestion
  const [isModalOpen, setIsModalOpen] = useState(false);
```
While it's not incorrect, a more conventional naming pattern is to prefix setter functions with set. matching the name of the state variable.